### ISSUE TITLE
StackOverflowError DuplexTcpClientPipelineFactory.peerWith

### ISSUE THREAD

**Issue by garyfeltham** [pjklauser/protobuf-rpc-pro#49]
```
java.lang.StackOverflowError
	at com.googlecode.protobuf.pro.duplex.client.DuplexTcpClientPipelineFactory.peerWith(DuplexTcpClientPipelineFactory.java:143)
	at com.googlecode.protobuf.pro.duplex.client.DuplexTcpClientPipelineFactory.peerWith(DuplexTcpClientPipelineFactory.java:143)```

Fix is

return peerWith(host,  port, null, null);

**Comment by pjklauser** [comment]
thanks for finding this embarrassing bug:)

**Comment by garyfeltham** [comment]
No worries, just running some testing and realised I introduced a bug should really be 
return peerWith(host,  port, bootstrap, null);

:)

**Comment by pjklauser** [comment]
well spotted. i corrected it now. Will be released in the next minor version.

**Comment by pjklauser** [comment]
released in 3.3.4
